### PR TITLE
Fixes data migration changes and daily extract changes.

### PR DESCRIPTION
**Pull request by ddebella**

### CODE REVIEW

File: migrations/app/20171204150838_app_reductions_add_audit_trail.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+exports.up = function (knex, Promise) {
+  var sql = 'ALTER TABLE app.reductions ADD updated_date DATETIME NOT NULL CONSTRAINT pdated_date_default DEFAULT GETDATE();'
```
Comment (davidmck95) [inline migrations/app/20171204150838_app_reductions_add_audit_trail.js]: missing the 'u' on updated_date_default

File: app/services/data/get-arms-totals.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const knex = require('knex')(knexConfig)
 module.exports = function (omKey, teamCode) {
   var whereObject = {
     'assessment_staff_key': omKey,
-    'assessmentent_team_key': teamCode // typo in extract
+    'assessment_team_key': teamCode // typo in extract
```
Comment (davidmck95) [inline app/services/data/get-arms-totals.js]: can we get get rid of this comment now if the typo has been fixed on this and in extract?

File: migrations/staging/20171204143053_stg_arms_update_column_name.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+exports.up = function (knex, Promise) {
+  var sql = 'EXEC sp_rename \'staging.arms.assessmentent_team_key\', \'assessment_team_key\';'
```
Comment (davidmck95) [inline migrations/staging/20171204143053_stg_arms_update_column_name.js]: Do you need to specify a third argument here, 'COLUMN'?
https://docs.microsoft.com/en-us/sql/relational-databases/system-stored-procedures/sp-rename-transact-sql

File: app/services/data/get-arms-totals.js (hunk 2 above)
Comment (ddebella) [reply app/services/data/get-arms-totals.js]: Well, we are leaving this as the change is in a migration file and not directly changed in this file.

File: migrations/app/20171204150838_app_reductions_add_audit_trail.js (hunk 1 above)
Comment (ddebella) [reply migrations/app/20171204150838_app_reductions_add_audit_trail.js]: 👍

File: migrations/staging/20171204143053_stg_arms_update_column_name.js (hunk 3 above)
Comment (ddebella) [reply migrations/staging/20171204143053_stg_arms_update_column_name.js]: The third parameter is for a data type change, we are not changing the type just the name.

**Pull request merged**